Empty.bot

```diff
@@ -0,0 +1,17 @@
+// Fills an empty room
+// Also demonstrates and tests most language features
+ _      1  |  #      #  |  #      # // This is a comment
```

Comment:
This is awesome! We never quite got to it, and frankly I'm not positive how we'd do it with packrat parsing